pkg/serverless/trace/trace_test.go

```diff
@@ -64,3 +65,20 @@ func TestStartEnabledTrueValidConfigValidPath(t *testing.T) {
 	assert.NotNil(t, agent.cancel)
 
 }
+
+func TestLoadConfigShouldBeFast(t *testing.T) {
+	timeout := time.After(100 * time.Millisecond)
```

Comment:
Timing-based tests are always annoying, generating false positives.  I know it seems like 100ms is forever, but somehow in CI things can take this long quite frequently.

Unless there's a way to test this without resorting to timing, I'd suggest setting this timeout to something quite large (5s?) since the timeout doesn't happen in the success case.

Reply:
The aim of the test is to detect that we're not trapped anymore in the 2s timeout due to grpc client failure so 5s won't make it. I've changed it to 1s